src/runtime/contributions-flow-test.js

```diff
@@ -83,6 +83,32 @@ describes.realWin('ContributionsFlow', {}, (env) => {
     await contributionsFlow.start();
   });
 
+  it('should allow for non-closable dialogs', async () => {
```

Comment:
optional: I have a slight preference for more concise test descriptions that start directly with the relevant verb rather than "should", so this test description would read "allows non-closable dialogs". 

I don't feel strongly about it though. I noticed the surrounding tests are all written in the "should..." style, so it seems fine to be consistent, too.

Reply:
That's definitely clearer! I updated all the surrounding test names